docs/kb/perftune-modes-sync.rst

```diff
@@ -2,7 +2,7 @@
 Updating the Mode in perftune.yaml After a ScyllaDB Upgrade
 ==============================================================
 
-In versions 5.1 (ScyllaDB Open Source) and 2022.2 (ScyllaDB Enterprise), we improved ScyllaDB's performance by `removing the rx_queues_count from the mode 
```

Comment:
I think this change is wrong. We do this change in those specific old versions. Either you can consider this so old that we no longer even have have this entire paragraph, or if you do leave it, you need to mention this antique version regarding the "upgrade" that this paragraph is about.